packages/error-reporting/src/google-apis/auth-client.js

```diff
@@ -127,7 +119,9 @@ class RequestHandler extends common.Service {
     if (this._config.getShouldReportErrorsToAPI()) {
       this.request({
         uri: 'events:report',
-        qs: RequestHandler.manufactureQueryString(this._config.getKey()),
+        qs: {
```

Comment:
This looks like it will use `{ key: null }` as `qs` if `this._config.getKey()` is `null` instead of using `null` for `qs` as was done previously.  I'm concerned that this could be introducing a bug.